src/output.F90

```diff
@@ -376,6 +376,16 @@ subroutine print_batch_keff()
     else
       write(UNIT=OUTPUT_UNIT, FMT='(23X)', ADVANCE='NO')
     end if
+      
+  end subroutine print_batch_keff
+
+!===============================================================================
+! PRINT_CMFD displays the CMFD related information to output after CMFD is 
+! executed. Will print blank line if CMFD is not on, to ensure consistent 
+! formatting of output columns
+!===============================================================================
+
+  subroutine print_cmfd()
```

Comment:
There's really no need to split up the `print_batch_keff` subroutine. Instead, I would just have it be called from `finalize_batch` (where you're currently calling `print_cmfd`).